README.md

```diff
@@ -16,13 +16,21 @@ usage in our [examples folder](./examples).
 Optimizely Agent is implemented in [Golang](https://golang.org/). Golang version 1.13+ is required for developing and compiling from source.
 Installers and binary archives for most platforms can be downloaded directly from the Go [downloads](https://golang.org/dl/) page.
 
-## Running from source
+## Running from source (Linux / OSX)
 Once Go is installed, the Optimizely Agent can be started via the following `make` command:
 ```bash
 make run
 ```
 This will start the Optimizely Agent with the default configuration in the foreground.
 
+## Running from source (Windows)
+A helper script is available under [scripts/build.ps1](./scripts/build.ps1) to automate compiling Agent in a Windows environment. Open a Powershell terminal and run 
+```bash
+Set-ExecutionPolicy -ExecutionPolicy Unrestricted -Scope CurrentUser
+.\scripts\build.ps1
+```
+The script will download and install both Git and Golang and then attempt to compile Agent. Click here to see expected [output](https://travis-ci.com/github/optimizely/agent) (look under Test/windows build.ps1 test)
```

Comment:
I don't think we need to link to travis for example output. I'd say remove this last sentence.